src/encoded/static/components/auditmatrix.js

```diff
@@ -233,7 +237,7 @@ class AuditMatrix extends React.Component {
                                         <div>
                                             <h3 style={{ marginTop: 0 }}>{context.title}</h3>
                                             <div>
-                                                <p>Click or enter search terms to filter the experiments included in the matrix.</p>
+                                                <p>Click or enter search terms to filter the {type} included in the matrix.</p>
```

Comment:
This is a lot of work for what seems like a one line, one word change.  Don't we just need to change 'experiments' to 'annotations' in the auditmatrix?

Reply:
Ignore per discussion, I miss understood the changes.